build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -602,6 +602,14 @@ class SeleniumWebDriverController {
     await this.driver.switchTo().frame(element);
   }
 
+  /**
+   * @param {number} index
+   * @return {!Promise}
+   */
+  async switchToFrameNum(index) {
+    await this.driver.switchTo().frame(index);
```

Comment:
Why didn't `switchToFrame` work with a handle to the `iframe` tag? I think this extra method makes the API more cluttered than it needs to be.

e.g. 
```
const swgOuterFrame = controller.findElement('iframe.some-classname-on-the-swg-iframe');
controller.switchToFrame(swgOuterFrame);
```